component/src/main/java/io/siddhi/extension/execution/time/UTCTimestampFunctionExtension.java

```diff
@@ -48,31 +50,65 @@
 @Extension(
         name = "utcTimestamp",
         namespace = "time",
-        description = "Function returns the system current time in UTC timezone with `yyyy-MM-dd HH:mm:ss` format.",
+        description = "Function returns the system current time in UTC timezone with given date format which " +
+                "defaults to `yyyy-MM-dd HH:mm:ss` if not provided.",
+        parameters = {
+                @Parameter(name = "date.format",
+                        description = "The format of the date value provided. " +
+                                "For example, `yyyy/MM/dd HH:mm:ss.SSS`.",
```

Comment:
Do we support yy/MM/dd. Else lts clearly state it here

Reply:
yes its any format user provides basically